index.html

```diff
@@ -147,22 +147,24 @@ <h3>Drop down to the API level</h3>
 
 <pre>
 <span class="comment"># use contents of a file to post a comment on issue #123 of the current repo</span>
-<span class=p></span><strong>hub api</strong> repos/{owner}/{repo}/issues/123/comments -F body=<span class=value>@mycomment.txt</span>
+<span class=p></span><strong>hub api</strong> repos/{owner}/{repo}/issues/123/comments --field body=<span class=value>@mycomment.txt</span>
 
 <span class="comment"># find a pull request that introduced a specific commit SHA into a repo</span>
 <span class=p></span>REPO="github/hub"
 <span class=p></span>SHA="b0db79db"
-<span class=p></span><strong>hub api graphql</strong> -t -f q="repo:<span class=value>$REPO</span> type:pr <span class=value>$SHA</span>" -f query=<span class=value>'
-  query($q: String!) {
-    search(query: $q, type: ISSUE, first: 3) {
-      nodes {
-        ... on PullRequest {
-          url
+<span class=p></span><strong>hub api graphql</strong> --flat \
+  --raw-field query="repo:<span class=value>$REPO</span> type:pr <span class=value>$SHA</span>" \
```

Comment:
Uhhh I don't like seeing `--field` and `--raw-field` in api examples; they are already verbose enough without that. I think I would like to keep `-F` and `-f` here and let the reader infer that they are fields.

Reply:
I agree, fixed.